docs/modules/operation/pages/quick-start/users.adoc

```diff
@@ -35,8 +35,6 @@ After logging in for the first time, make sure to change the default `admin` acc
 . Update the password and click *OK*.
 . Click *Finish* at the bottom of the *Modify User* screen to save changes.
 
-WARNING: You cannot use angle brackets (<>), single (') and double quotation marks ("), or the ampersand symbol (&) in the user ID.
-
 We recommend that you do not use the default `admin` user account, but instead xref:deep-dive/user-management/user-config.adoc#ga-user-config [create specific users] with the Admin role and other permissions.
```

Comment:
```suggestion
We recommend that you do not use the default `admin` user account, but instead xref:deep-dive/user-management/user-config.adoc#ga-user-config [create specific users] with the Admin role and other permissions.
For the purpose of the quick start, create one user for yourself, with admin permissions, and include your email address and any other contact information (for example, a mobile phone number).
You will use this user in <<quick-start/notifications.adoc, Step 2: Turn on Default Notifications>> of the quick start.
```

Reply:
I'm going to move the extra content here down, so that it doesn't break the flow of recommendations for the `admin` account.